deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/deployment.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: indexer
+spec:
+  template:
+    spec:
+      terminationGracePeriodSeconds: 600
+      containers:
+        - name: indexer
+          resources:
+            limits:
+              cpu: "10"
+              memory: 120Gi
+            requests:
+              cpu: "10"
+              memory: 120Gi
```

Let's start with minimal resources and see how it behaves first.

```suggestion
            limits:
              cpu: "3"
              memory: 10Gi
            requests:
              cpu: "3"
              memory: 10Gi
```